### PR TITLE
1291 auto dismiss toast manual dismiss behaviour

### PR DESCRIPTION
**Pull request by GCHQ-Developer-530**
## Summary of the changes
Update auto dismiss toast to turn manual when hovered over. Add empty description to auto dismiss
loading indicator so that "loading" isn't read out to a screen reader

BREAKING CHANGE: Behaviour change to auto dismiss toasts to show dismiss button when toast is
hovered over. May cause test breakages.

## Related issue
#1291 

### General 

- [x] Changes to docs package checked and committed.
- [x] All acceptance criteria reviewed and met. 

### Testing

- [x] Relevant unit tests and visual regression tests added. 
- [x] Visual testing against Figma component specification completed. 
- [x] Playground stories in React Storybook up to date, with any prop changes and additions addressed. 

### Accessibility 

- [x] Accessibility Insights FastPass performed.
- [x] A11y unit test added and yields no issues.
- [x] A11y plug-in on Storybook yields no issues. 
- [x] Manual screen reader testing performed using NVDA and VoiceOver. 
- [x] Manual keyboard testing for keyboard controls and logical focus order. 
- [x] Correct roles used and ARIA attributes used correctly where required. 
- [x] Logical heading structure is maintained, and the HTML elements used for headings can be changed to fit within the wider page structure. 

### Resize/zoom behaviour 

- [x] Page can be zoomed to 400% with no loss of content. 
- [x] Screen magnifier used with no issues. 
- [x] Text resized to 200% with no loss of content.
- [x] Text spacing increased as per the [WCAG 1.4.12 success criterion](https://www.w3.org/TR/WCAG21/#text-spacing) with no loss of content.

### System modes

- [x] Browser setting 'prefers reduced motion' tested. No animations or motion visible whilst this setting is on. 
- [x] Windows High Contrast mode tested with no loss of content.
- [x] System light and dark mode tested with no loss of content.
- [x] Browser support tested (Chrome, Safari, Firefox and Edge). 

### Testing content extremes

- [x] Min/max content examples tested with no loss of content or overflow. 
- [x] All prop combinations work without issue. 
- [x] Tested for FOUC (Flash of Unstyled Content) in both SSR (Server-Side Rendering) and SSG (Static Site Generation) settings.
- [x] Controlled and uncontrolled input components tested.

### CODE REVIEW

**Review by GCHQ-Developer-299 — DISMISSED** [review]

**Review by GCHQ-Developer-299 — DISMISSED** [review]

**Comment by ad9242** [comment]
a bit of strange issue using auto-dismiss example:

- click the button to show the toast
- tab so that the `x` appears and gets focus
- click elsewhere on page (or mouse over the toast & then off again) - it goes back to a countdown indicator
- now try to press `tab` to focus again - the `x` button never displays the focus indicator

**Review by gd2910 — APPROVED** [review]

File: packages/react/src/component-tests/IcAccordion/IcAccordion.cy.tsx
Comment (GCHQ-Developer-299) [inline packages/react/src/component-tests/IcAccordion/IcAccordion.cy.tsx:1]: Should this formatting change be included in the PR? It seems unrelated

Comment (GCHQ-Developer-530) [reply packages/react/src/component-tests/IcAccordion/IcAccordion.cy.tsx:1]: It was causing the CI to fail for prettier so I had to update, not sure why its only failing this PR though :(

**Review by GCHQ-Developer-299 — APPROVED** [review]

**Review by ad9242 — APPROVED** [review]
